### ISSUE TITLE
Update lint command: check for blurb in config.json

### ISSUE THREAD

**Issue by jonmcalder** [exercism/v2-configlet#40]
Based on the updates outlined in [discussions/173](https://github.com/exercism/discussions/issues/173), it seems as if the blurb in `config.json` will (later) be a required field (unless there is some fallback option?) and `configlet lint` should thus check for it.

**Comment by kytrinyx** [comment]
Yeah, the blurb is required. I don't think we can provide a good default for this one.

**Comment by jonmcalder** [comment]
The only (remotely) decent possibility as a fallback option might be a regular expression to extract the first sentence (or 2) from `ABOUT.md`. Not great though, so better to make it required.

**Comment by kytrinyx** [comment]
Yeah, I really want to be super-duper careful about making the blurbs well-crafted.

Maybe we can have an `@exercism/copywriting` team of volunteers who are willing to help craft the docs and the track-specific copy, including getting extra eyes on maintainer bios and things.

**Comment by robphoenix** [comment]
As the lint command wouldn't necessarily be able to make any judgement calls on the quality/validity of the blurb, would it instead check to ensure the blurb is not an empty string, and no longer than N characters? 
So the linting is just ensuring it exists and is within specified limits, and a it would then be for human eyes to ensure it's well-crafted?
Or would there be more to it?

**Comment by kytrinyx** [comment]
> So the linting is just ensuring it exists and is within specified limits, and a it would then be for human eyes to ensure it's well-crafted?

Yepp, that's exactly what I was thinking. We might add a bot to ping the hypothetical copywriting team when the blurb or docs change, but for the linter, just the presence of it and within limits. We need to figure out what those limits are. @iHiD 30-ish words... are we operating with standard morse word length of "PARIS" (five characters, the actual duration of each letter obviously doesn't matter here).

**Comment by nywilken** [comment]
> Yepp, that's exactly what I was thinking. We might add a bot to ping the hypothetical copywriting team when the blurb or docs change, but for the linter, just the presence of it and within limits. We need to figure out what those limits are. @iHiD 30-ish words... are we operating with standard morse word length of "PARIS" (five characters, the actual duration of each letter obviously doesn't matter here).

I started looking into this issue. Is 30 words (5 characters) a bit much? Looking at some of the existing blurbs they are around 10 - 16 words. With that said should this rule apply to maintainer bios as well if show on website property is set to true?
